.buildkite/bk.integration.pipeline.yml

```diff
@@ -277,7 +277,7 @@ steps:
         matrix:
           setup:
             variants:
-            - "basic,wolfi,complete,complete-wolfi,service,cloud"
+            - "basic,wolfi,complete,complete-wolfi,service,elastic-otel-collector-wolfi,elastic-otel-collector"
```

Comment:
This is missing slim and slim-wolfi?

Reply:
We need to smoke test it somehow to make sure it isn't completely broken. If there's a better way that's great but if this is the only way then we need it.

Reply:
at the time being I cannot offer a better solution than introducing these variants in the k8s integration tests. However, I split up the variants into two groups (non-wolfi and wolfi ones) so k8s integration tests duration won't be hindered

Reply:
ok `slim` and `slim-wolfi` variants are not being tested under both kustomize and helm chart k8s integration tests. Feel free to unresolve if you spot anything else 🙂